server/plugins/FeatureDetection.cpp

```diff
@@ -419,9 +431,148 @@ void RunningSum_next_k( RunningSum *unit, int inNumSamples )
 	unit->msum2 =  sum2;
 }
 
+
+// RunningSum2 adds a variable window size to RunningSum - mtmccrea
+void RunningSum2_Ctor( RunningSum2* unit )
```

Comment:
i'm not seeing any initial setting of `OUT0(0)`, so this will output an initial sample of garbage memory.